src/db/timezones.ts

```diff
@@ -12,18 +12,34 @@ async function addTimeZone(timezone: String, userID: String) {
   await userObject.save()
 }
 
-async function compareTimezones(user1: String, user2: String) {
-  // Find the users
-  const user1Object: any = await userModel.findOne({ id: user1 })
-  const user2Object: any = await userModel.findOne({ id: user2 })
+async function compareTimezones(user1Id: String, user2Id: String) {
+  // Find both users
+  const [user1Object, user2Object] = await Promise.all([
+    userModel.findOne({ id: user1Id }),
+    userModel.findOne({ id: user2Id }),
+  ])
 
   if (!user1Object || !user2Object) {
+    return "One or both of the users is not verified"
+  }
+
+  if (!user1Object?.timezone || !user2Object?.timezone) {
     return "One or both of the users don't have a timezone set"
   }
 
-  // I need help with comparing the timezones
-  // timezones are tored like this: UTC+1 or UTC-1 in the database
-  // I need to compare the timezones and return the difference
+  // Comparing Timezones
+  const user1 = new Date(
+    `Sun, 30 May 2021 14:59:15 ${user1Object.timezone}`
+  ).getTime()
+
+  const user2 = new Date(
+    `Sun, 30 May 2021 14:59:15 ${user2Object.timezone}`
+  ).getTime()
+
+  const Status = user1 < user2 ? "ahead" : "behind"
```

Comment:
Why "Status" with an uppercase ?
```suggestion
  const status = user1 < user2 ? "ahead" : "behind"
```
You also have to modify the return statement